internal/usecase/interfaces/scene.go

```diff
@@ -29,7 +39,7 @@ type Scene interface {
 	Create(context.Context, id.ProjectID, *usecase.Operator) (*scene.Scene, error)
 	AddWidget(context.Context, id.SceneID, id.PluginID, id.PluginExtensionID, *usecase.Operator) (*scene.Scene, *scene.Widget, error)
 	UpdateWidget(context.Context, UpdateWidgetParam, *usecase.Operator) (*scene.Scene, *scene.Widget, error)
-	RemoveWidget(context.Context, id.SceneID, id.PluginID, id.PluginExtensionID, *usecase.Operator) (*scene.Scene, error)
+	RemoveWidget(context.Context, id.SceneID, id.PluginID, id.PluginExtensionID, *scene.WidgetLocation, *usecase.Operator) (*scene.Scene, error)
```

Comment:
If you follow the review comments I wrote in the GraphQL schema, you will need to fix them as well.